src/plugins/data/public/ui/query_editor/query_editor_top_row.tsx

```diff
@@ -246,33 +243,12 @@ export default function QueryEditorTopRow(props: QueryEditorTopRowProps) {
           getQueryStringInitialValue={getQueryStringInitialValue}
           persistedLog={persistedLog}
           dataTestSubj={props.dataTestSubj}
-          queryEditorHeaderRef={queryEditorHeaderRef}
-          queryEditorBannerRef={queryEditorBannerRef}
+          queryLanguage={queryLanguage}
         />
       </EuiFlexItem>
     );
   }
 
-  function renderQueryEditorExtensions() {
```

Comment:
this is a good clean up. that i need to do after restructuring. I have a clean up task in https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6957 but some of the render functions i added are in the wrong location and that's cause i tried to not touch query string input. so thanks for starting the clean up!